ci-scripts/validate-translations.sh

```diff
@@ -35,14 +32,35 @@ check_import_statements() {
   done
 }
 
-# Ensure the translations file exists
-[ -f "$translations_file" ] || { echo "Translations file not found: $translations_file"; exit 1; }
+# Function to process a translation folder
+process_translation_folder() {
+  local folder_path=$1
+  local translations_file="$folder_path/translations.ts"
+
+  # Ensure the translations file exists
+  [ -f "$translations_file" ] || { echo "Translations file not found: $translations_file"; exit 1; }
+
+  # Get the list of translation folders
+  translation_folders=$(find "$folder_path" -mindepth 1 -maxdepth 1 -type d -exec basename {} \;)
+
+  # Check if each folder and index file is listed in the translations file
+  for folder in $translation_folders; do
+    check_export_statements "$folder" "$translations_file"
+    check_import_statements "$folder"
+  done
+}
+
+# Check if folder_path is provided as an argument
+if [ -z "$1" ]; then
+  # Find all translation folders in the repository
+  translation_paths=$(find . -type d -path '*/assets/translations')
```

Comment:
While running the script on local (where having `/dist` folder with already built libs), I've got a failure of the script:
`Translations file not found: ./dist/organization/account-summary/assets/translations/translations.ts`

```suggestion
  # Find all translation folders in the repository, excluding node_modules/ and dist/
  translation_paths=$(find . -type d -path '*/assets/translations' -not -path './node_modules/*' -not -path './dist/*')
``